### ISSUE TITLE
Find a better picture for replacing third image of the carousel.

### ISSUE THREAD

**Issue by ebonilla91** [PinoleroMedia/coming-soon-website#4]
The logo in the third image of the Carousel is distorted

**Comment by Luis-Palacios** [comment]
Yes the background image also may not be the best, feel free to change it too

**Comment by Luis-Palacios** [comment]
![logo-pinoleromedia-portada](https://cloud.githubusercontent.com/assets/4844997/9451171/3ba44648-4a6a-11e5-8ddb-9a1438b1bb72.png)
